addons/fleet/data/fleet_cars_data.xml

```diff
@@ -1,492 +1,378 @@
-          <field name="vehicle_type">car</field>
-      </record>
-      <record id="model_classgl" model="fleet.vehicle.model">
-          <field name="name">Class GL</field>
-          <field name="brand_id" ref="brand_mercedes" />
-          <field name="vehicle_type">car</field>
-      </record>
-      <record id="model_classglk" model="fleet.vehicle.model">
-          <field name="name">Class GLK</field>
-          <field name="brand_id" ref="brand_mercedes" />
-          <field name="vehicle_type">car</field>
-      </record>
-      <record id="model_classr" model="fleet.vehicle.model">
-          <field name="name">Class R</field>
-          <field name="brand_id" ref="brand_mercedes" />
-          <field name="vehicle_type">car</field>
-      </record>
-      <record id="model_classs" model="fleet.vehicle.model">
-          <field name="name">Class S</field>
-          <field name="brand_id" ref="brand_mercedes" />
-          <field name="vehicle_type">car</field>
-      </record>
-      <record id="model_classslk" model="fleet.vehicle.model">
-          <field name="name">Class SLK</field>
-          <field name="brand_id" ref="brand_mercedes" />
-          <field name="vehicle_type">car</field>
-      </record>
-      <record id="model_classsls" model="fleet.vehicle.model">
-          <field name="name">SLS</field>
-          <field name="brand_id" ref="brand_mercedes" />
-          <field name="vehicle_type">car</field>
-      </record>
+        <field name="image_1920" type="base64" file="fleet/static/img/brand_suzuki-image.png"/>
+    </record>
+
+    <record id="model_category_sedan" model="fleet.vehicle.model.category">
+        <field name="name">Sedan</field>
+    </record>
+    <record id="model_category_estate" model="fleet.vehicle.model.category">
+        <field name="name">Estate</field>
+    </record>
+    <record id="model_category_compact" model="fleet.vehicle.model.category">
+        <field name="name">Compact</field>
+    </record>
+    <record id="model_category_suv" model="fleet.vehicle.model.category">
+        <field name="name">SUV</field>
+    </record>
+    <record id="model_category_coupe" model="fleet.vehicle.model.category">
+        <field name="name">Coupe</field>
+    </record>
+    <record id="model_category_convertible" model="fleet.vehicle.model.category">
+        <field name="name">Convertible</field>
+    </record>
+    <record id="model_category_mpv" model="fleet.vehicle.model.category">
+        <field name="name">MPU</field>
+    </record>
+    <record id="model_category_bmx" model="fleet.vehicle.model.category">
+        <field name="name">BMX</field>
+    </record>
+    <record id="model_category_vtt" model="fleet.vehicle.model.category">
+        <field name="name">VTT</field>
+    </record>
+    <record id="model_category_city" model="fleet.vehicle.model.category">
+        <field name="name">City</field>
+    </record>
+
+    <record id="model_corolla_ts" model="fleet.vehicle.model">
```

Comment:
It's a good move to drop all the data vehicle models, but why adding this as new data ? Probably unrelated to end users need + nobody is gonna maintain this, resulting into outdated data in a few versions.